src/test/java/uk/gov/ons/ctp/common/domain/EstabTypeTest.java

```diff
@@ -45,5 +45,6 @@ public void unknownTurnsIntoOther() {
     assertEquals(EstabType.OTHER, otherEstabType);
     // and that OTHER addressType is empty optional
     assert (otherEstabType.getAddressType().isEmpty());
+    assert (otherEstabType.getCode().equals("OTHER"));
```

Comment:
Addition of this seems a bit superfluous? The previous statement _assertEquals(EstabType.OTHER, otherEstabType);_ checks the enum is of type OTHER and has all the associated properties, i.e. code is going to be "OTHER". Maybe remove as redundant code?

Reply:
done